### PR TITLE
pushing up

### PR DESCRIPTION
**Pull request by andrewbastian**

### CODE REVIEW

**Comment by liam-mei** [comment]
I just looked at it.  It's got a good skeleton, but it seems to be missing a few pieces.

https://htmlpreview.github.io/?https://github.com/andrewbastian/User-Interface/blob/andrew-bastian/great-idea-website/index.html

is what it looks like right now.  Is it possible you missed a commit before pushing?  It appears only the top half was completed.

**Comment by Derrick-Mei** [comment]
It's looking pretty good, but it looks like it's missing a few pieces.  Is it possible that you missed a commit before pushing?

This is what it looks like right now:

https://htmlpreview.github.io/?https://github.com/andrewbastian/User-Interface/blob/andrew-bastian/great-idea-website/index.html

Good job on the hover, but I think you missed the bottom half of the page.

**Comment by andrewbastian** [comment]
I did mess up some part of my last push. Sorry. this update is as far as I was able to get on this project. Still some issues with the CSS.

File: great-idea-website/index.html [hunk 1]
```diff
@@ -15,49 +15,87 @@
...
-  On
-  Demand
-
-  Get Started
-
-  <img src="img/header-img.png" alt="Image of a code snippet.">
-
-  Features
+    <header>
+    <div class="container">
```
Comment (liam-mei) [inline great-idea-website/index.html:19]: Over here you have a header tag as well as in inner div.  I would say one of them is redundant as it doesn't provide any additional functionality.

Also, conventionally the container class is used to targeting the div/body that is wrapping the page and not the header.  I would name the container class something else, like navbar.

File: great-idea-website/index.html [hunk 2]
```diff
@@ -15,49 +15,87 @@
...
+      <nav>
+    <a href="#"> Services </a>
+    <a href="#">Product</a>
+    <a href="#"> Vision</a>
+    <a href="#">Features</a>
+    <a href="#">About</a>
+    <a href="#">Contact</a>
+        </nav>
```
Comment (liam-mei) [inline great-idea-website/index.html:27]: The indentation can be a bit better.  Proper indentation makes reading code a lot easier.  Pro-Tip.  If you right click on your html page inside VSC you should have an option to format code.  It's amazing

File: great-idea-website/index.html [hunk 3]
```diff
@@ -15,49 +15,87 @@
...
+      On</br>
+      Demand</br>
+</h1>
+    <button>Get Started</button>
+  </div>
+  <img class = cta-img src="img/header-img.png" alt="Image of a code snippet."/>
+
+</section>
+
+<hr></hr>
```
Comment (liam-mei) [inline great-idea-website/index.html:45]: Good job with the `<hr>` tag for the break!  I think it's the first time I've seen that tag

File: great-idea-website/index.html [hunk 4]
```diff
@@ -15,49 +15,87 @@
...
   Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+</p>
+</div>
+</section>
 
 
-  About
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
+<div>
```
Comment (liam-mei) [inline great-idea-website/index.html:64]: This outer div wrapping the img is not necessary.  A img itself it an html element and can be targeted using regular methods as you are already doing.

File: great-idea-website/index.html [hunk 5]
```diff
@@ -15,49 +15,87 @@
...
+
 
-  Contact
-  123 Way 456 Street
-  Somewhere, USA
-  1 (888) 888-8888
-  sales@greatidea.io
+<footer>
+  Contact</br>
+  123 Way 456 Street</br>
```
Comment (liam-mei) [inline great-idea-website/index.html:93]: did you know there's a HTML tag for `<address>` ?  I learned that yesterday.

File: great-idea-website/css/index.css [hunk 6]
```diff
@@ -63,4 +64,137 @@ h1, h2, h3, h4, h5 {
...
+    width: 300px;
+}
+
+.SPV{
+	width: 100%;
+    height: 166px;
+    text-align: center;
+    width: 100%;
+}
+.services, .product, .vision{
```
Comment (liam-mei) [inline great-idea-website/css/index.css:175]: great job using the syntax for applying multiple stylings to the same classname.  That being said though, considering that they are all doing the same thing, I would personally just use one generic class like .bottom-text?  It's a totally a personal design choice, but I find that a more simplistic approach is easier to read.

**Review by liam-mei — APPROVED** [review]

**Comment by liam-mei** [comment]
**MVP**
- [x] Inline-block and box model was used to create page (no flexbox or css grid)
- [x] page was successfully built and looks like design file
--------------------------------------------------------------------------

**Additional Comments**
- [x]  Follows Lambda Git Flow
- [ ]  Frequent, clear, and descriptive commits
- [ ]  Code is organized and easy to read
- [x] Code follows consistent naming conventions, language idioms, and style.
- [ ] code is DRY (Don't Repeat Yourself)

--------------------------------------------------------------------------

Score: 2

Overall I'm really pleased with the HTML structure of your project.  Other than a few redundant divs, I think your HTML is the best in the group.  

The CSS styling is getting there.  All the important pieces to make the arrangement work is good, but individual pieces could be improved such as some margin and font size.  The naming of your CSS is also really good.  They're descriptive and short, some I might just combine into one css class though.

Git:
Frequent commits are important for a good developer.  I can't grade on that yet as we haven't gone over it yet, but I'd like to see at least 6 descriptive commits on your next project.

**Pull request merged**